pkgs/applications/audio/cava/default.nix

```
@@ -27,7 +29,7 @@ stdenv.mkDerivation rec {
     libpulseaudio
     ncurses
     iniparser
-  ];
+  ]++ lib.optionals withSDL2 [SDL2];
```

```suggestion
  ]++ lib.optional withSDL2 SDL2;
```
Doesn't matter much to me, maybe a committer has a stronger opinion on this.